backend/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:13.14-alpine
```

Comment:
Absolutely, the thing is that the dev proxy did not properly work. Because of that and because we didn't want to waste more time on this we left it with hard coded urls